pages/Berries/jaboca.md

```diff
@@ -0,0 +1,49 @@
+# Jaboca
+
+Jaboca is a type of [Berry](https://wiki.pokeclicker.com/) that can be planted in the [Farm](https://wiki.pokeclicker.com/). It can be obtained by having at least one empty plot and at least 600 points of bitter on the field. The [Durin](https://wiki.pokeclicker.com/) must be unlocked to unlock this berry. Various [Farming Setups](https://wiki.pokeclicker.com/) using Jaboca can be found [here](https://wiki.pokeclicker.com/).
```

Comment:
Updated to also allow single links too:
`[[Berries]]` to link to the main Berries page or `[[Berries/Cheri]]` to link to the Cheri page.
```suggestion
Jaboca is one of the types of [[Berries]] that can be planted in the [Farm](https://wiki.pokeclicker.com/). It can be obtained by having at least one empty plot and at least 600 points of bitter on the field. The [Durin](https://wiki.pokeclicker.com/) must be unlocked to unlock this berry. Various [Farming Setups](https://wiki.pokeclicker.com/) using Jaboca can be found [here](https://wiki.pokeclicker.com/).
```